lessons/day_05/python/resolvers.py

```diff
@@ -0,0 +1,127 @@
+from ariadne import QueryType, ObjectType, EnumType, MutationType
+from random import randint
+from models import Skill, Person, person_friends, person_skills
```

Comment:
remove unused import 
```suggestion
from models import Skill, Person
```

Reply:
Remember that the IDs it not integer, but a string,